app/helpers/breadcrumb_helper.rb

```diff
@@ -0,0 +1,29 @@
+module BreadcrumbHelper
+  def breadcrumb_part_for_previous_page
+    path, title = path_and_title_from_referer
+
+    content_tag(:li) do
+      content_tag(:a, href: path) do
+        title
+      end
+    end
+  end
+
+  private
+
+  def path_and_title_from_referer
+    if request_uri.start_with?(search_appointments_path)
+      [request_uri, 'Search']
+    elsif request.start_with?(activities_path)
```

Comment:
@AndrewVos This line is causing an error on master for me (request_uri not request?)